<launch>
  <include file="$(find bender_arm_control)/launch/l_arm.launch"/>
  <node pkg="bender_arm_control" type="motor_monitor.py" name="l_arm_error_monitor" output="screen">
    <param name="monitor_type" value="error" />
    <param name="rate" value="10.0" />
    <rosparam param="motor_ids">[7,8]</rosparam>
    <rosparam param="motor_names">['ext','int']</rosparam>
    <remap from="motor_states" to="/bender/motor_states/l_arm_port"/>
 </node>
 <node pkg="bender_arm_control" type="motor_monitor.py" name="l_arm_load_monitor" output="screen">
    <param name="monitor_type" value="load" />
    <param name="rate" value="10.0" />
    <rosparam param="motor_ids">[7,8]</rosparam>
    <rosparam param="motor_names">['ext','int']</rosparam>
    <remap from="motor_states" to="/bender/motor_states/l_arm_port"/>
 </node>
</launch>